app/src/main/res/layout-land/topic_info_fragment.xml

```diff
@@ -72,7 +73,7 @@
         android:id="@+id/download_status_image_view"
```

Comment:
Margin End in `download_size_text_view` with be `android:layout_marginEnd="72dp"`

Reply:
@chrk2205 This has still not been updated.